packages/frontend/src/pages/Spaces.tsx

```diff
@@ -79,27 +84,32 @@ const Spaces: FC = () => {
                     )}
                 </Group>
 
-                <ResourceView
-                    view={ResourceViewType.GRID}
-                    items={wrapResourceView(
-                        spaces.map(spaceToResourceViewItem),
-                        ResourceViewItemType.SPACE,
-                    )}
-                    headerProps={{
-                        title: 'Spaces',
-                    }}
-                    emptyStateProps={{
-                        icon: <IconFolders size={30} />,
-                        title: 'No spaces added yet',
-                        action:
-                            !isDemo && userCanManageSpace ? (
-                                <Button onClick={handleCreateSpace}>
-                                    Create space
-                                </Button>
-                            ) : undefined,
-                    }}
-                    pinnedItemsProps={{ projectUuid, pinnedListUuid: '' }}
-                />
+                <PinnedItemsProvider
+                    projectUuid={projectUuid}
+                    organizationUuid={user.data?.organizationUuid ?? ''}
+                    pinnedListUuid={project.data?.pinnedListUuid ?? ''}
+                >
```

Comment:
this is not necessary since Spaces page does not have a reorder enabled and we also get if space is pinned or not from the data that is inside `spaces`

Reply:
inside `ResourceViewGrid` you use `usePinnedItemsContext` which throws an error if you don't have a `PinnedItemsProvider`. we can `usePinnedItemsContext` return undefined and not throw an error instead